crates/noirc_frontend/src/parser/mod.rs

```diff
@@ -371,19 +371,15 @@ impl ForRange {
                     expression: array,
                 });
 
-                let ident = |name: &str| Ident::new(name.to_string(), array_span);
-
-                // std::array::len(array)
+                // array.len()
                 let segments = vec![array_ident];
                 let array_ident =
                     ExpressionKind::Variable(Path { segments, kind: PathKind::Plain });
 
-                let segments = vec![ident("std"), ident("array"), ident("len")];
-                let func_ident = ExpressionKind::Variable(Path { segments, kind: PathKind::Dep });
-
-                let end_range = ExpressionKind::Call(Box::new(CallExpression {
-                    func: Box::new(Expression::new(func_ident, array_span)),
-                    arguments: vec![Expression::new(array_ident.clone(), array_span)],
+                let end_range = ExpressionKind::MethodCall(Box::new(MethodCallExpression {
+                    object: Expression::new(array_ident.clone(), array_span),
+                    method_name: Ident::new("len".to_string(), array_span),
```

Comment:
If we changed the method name from `len` to `length`, is there a place that we could define the constant for that, so we don't have a string literal?

Reply:
Not really, we could move this literal into a constant (there are no other instances of it), but you would still have to change multiple files since you'd have to update array.nr in the noir stdlib to change the len method name there as well.